mission_yaml/SEA45_M48.yml

```diff
@@ -310,37 +309,44 @@ netcdf_variables:
     units:        dbar
     coordinates:  time depth latitude longitude
 
+  ad2cp_altitude:
```

Comment:
While merging I accidentally re-added the ad2cp_altitude variable, could you remove this adain?